src/github/activityBarViewProvider.ts

```diff
@@ -113,6 +115,18 @@ export class PullRequestViewProvider extends WebviewViewBase implements vscode.W
 		}
 	}
 
+	private reRequestReview(message: IRequestMessage<string>): void {
```

Comment:
Copied from https://github.com/joshuaobrien/vscode-pull-request-github/blob/63ef3f541c6ad96e7eba60fec06da0fb0f52bbae/src/github/pullRequestOverview.ts#L891

Reply:
There's likely a lot of duplication between these two files. I'll take it as a debt item to clean them up.